.github/workflows/bundle_with_dakota_caller.yml

```diff
@@ -11,6 +11,10 @@ env:
 
 on:
   pull_request:
+  push:
+    branches:
+      - main
+    tags: "*"
```

Comment:
Don't we want to limit this to specific tags? E.g. "v*" ? This to avoid any dev-tags or similar?

Reply:
Added v* glob for this